build/webpack.styles.js

```diff
@@ -1,35 +1,61 @@
 import ExtractTextPlugin from 'extract-text-webpack-plugin'
-// import postcssPlugins from './webpack.postcss'
 
-const loaderOptions = [
-  { loader: 'css-loader', query: { modules: true, importLoaders: 1 } },
-  { loader: 'postcss-loader'
-    // pending new postcss-loader release - then remove loaderOptionsPlugin
-    // options: {
-    //   plugins: function () {
-    //     return postcssPlugins
-    //   }
-    // }
+const style = 'style-loader'
+const css = 'css-loader'
+const postcss = 'postcss-loader'
+const sass = 'sass-loader'
+
+const cssOptions = [{
+  loader: css,
+  query: {
+    modules: true,
+    importLoaders: 1
   }
+}, {
+  loader: postcss
+}]
+
+const sassOptions = [
+  { loader: css, query: { modules: false, importLoaders: 1 } },
+  { loader: postcss },
+  { loader: sass, query: { outputStyle: 'compressed', includePaths: /node_modules/ } }
 ]
 
 const styleLoaders = {
-  development: {
-    test: /\.css$/,
-    loaders: [
-      'style-loader',
-      ...loaderOptions
-    ]
-  },
-  production: {
-    test: /\.css$/,
-    loaders: ExtractTextPlugin.extract({
-      fallbackLoader: 'style-loader',
-      loader: [
-        ...loaderOptions
+  development: [
+    {
+      test: /\.css$/,
+      use: [
+        style,
+        ...cssOptions
       ]
-    })
-  }
+    }, {
+      test: /\.scss$/,
+      use: [
+        style,
+        ...sassOptions
```

Comment:
you mean development and production? In development, we don't want to extract all the styles because it adds unnecessary overhead and build time on each recompile...so when we're in development, we just want to use the style-loader to inject all the styles into the <head> of the HTML file. This is exponentially faster.

For production, eventually we want to get to the point of chunking our CSS down to individual CSS files and only loading the CSS we need on a given route....or we may opt to use inline JS styles - which if we go that direction - would enable us to remove the extract-text-webpack-plugin entirely since we wouldn't need to compile css files for production.